package.json

```diff
@@ -166,6 +166,8 @@
     "electron-debug": "^1.0.1",
     "eslint-formatter-pretty": "^1.1.0",
     "eslint-plugin-react": "^6.9.0",
+    "formsy-material-ui": "github:jolan/formsy-material-ui",
```

Comment:
Yeah I was hoping a release would happen but it hasn't yet:

https://github.com/mbrookes/formsy-material-ui/issues/190

I can downgrade to the last released version and rip out some of the new features.

BTW, we have this too:

"react-tap-event-plugin": "git+https://github.com/zilverline/react-tap-event-plugin.git#master"